test/test_nn.py

```diff
@@ -13051,7 +13051,7 @@ def test_ConvTranspose2d_large_output_padding(self, device, dtype):
 
     @onlyCUDA
     @tf32_on_and_off(0.01)
-    @dtypes(*ALL_TENSORTYPES)
+    @dtypes(*get_all_fp_dtypes(include_bfloat16=False))
```

Comment:
This is close but not exactly what we want. A few things:

- don't use the `get_all` macros, either
- preserve the bfloat16 dtype when testing on ROCm

The dtypes should either be listed explicitly or you can use one of the functions styled after the ATen dispatch macros which are imported from here: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_dtype.py and defined like this: 

https://github.com/pytorch/pytorch/blob/4f35b9144cd3b84779139f8946d579b52bf4c1e0/torch/testing/_legacy.py#L64

Reply:
> A few things:
> 
> - don't use the get_all... macros, either
> - preserve the bfloat16 dtype when testing on ROCm

Sure, I will keep it in mind. Thanks :)

> The dtypes should either be listed explicitly or you can use one of the functions styled after the ATen dispatch macros

Thanks for the hint. I've updated the files. Will you please take a look?